Functions/Administration/xASL_adm_GetPhilipsScaling.m

```diff
@@ -29,45 +30,59 @@
 rescaleSlopeNifti = header.dat.scl_slope;
 
 if isfield(parms,'RWVSlope')
-	% If the RealWorldValue is present, then dcm2nii scales to them and ignores everything else
-    fprintf('=========================================== Scaling ==========================================\n');
-	if length(parms.RWVSlope)>1
-		[~,idx] = min(abs(parms.RWVSlope - rescaleSlopeNifti));
-		parms.RWVSlope = parms.RWVSlope(idx);
-		if isfield(parms,'RescaleSlopeOriginal') && length(parms.RescaleSlopeOriginal)>1
-			parms.RescaleSlopeOriginal = parms.RescaleSlopeOriginal(idx);
+	% Obtains dcm2niix version used for conversion and ExploreASL version
+	% If ExploreASL version >= 1.10.0 or dcm2niix version >= 20210317
+	if isfield(parms,'ConversionSoftware') && (strcmp(parms.ConversionSoftware,'dcm2nii') || strcmp(parms.ConversionSoftware,'dcm2niix')) && isfield(parms,'ConversionSoftwareVersion')
+		dcm2niixVersion = parms.ConversionSoftwareVersion;
+		dcm2niixVersion = str2num(dcm2niixVersion(end-7:end));
+	else
+		dcm2niixVersion = 0;
+	end
+	
+	% It has to use the correct dcm2niix version that is able to handle this parameter and it also needs to report that it did do that
+	if dcm2niixVersion >= 20210317 && isfield(parms,'UsePhilipsFloatNotDisplayScaling') && (parms.UsePhilipsFloatNotDisplayScaling == 1)
```

Comment:
`>=` is correct. Because starting with this version, the bug was fixed. So all newer versions handle it correctly, that's why we need `>=` and not `==`.